feat(cwts): add Choose what to sync to fx_desktop_v2

vladikoff opened the pull request:
Fixes #3213

vladikoff commented:
Ref https://bugzilla.mozilla.org/show_bug.cgi?id=1218022

vladikoff commented:
@shane-tomlinson this should be ready to go!

shane-tomlinson commented on tests/functional/firstrun_sign_up.js:
```diff
@@ -59,6 +59,33 @@ define([
+        .end()
+
+        .findByCssSelector('button[type=submit]')
+        .click()
```
nit: indent the click.

shane-tomlinson commented on tests/functional/firstrun_sign_up.js:
```diff
@@ -59,6 +59,33 @@ define([
         .then(FunctionalHelpers.testIsBrowserNotified(self, 'fxaccounts:can_link_account'))
+
+        // user should be transitioned to the choose what to Sync page
+        .findByCssSelector('#fxa-choose-what-to-sync-header')
```
Is Fx Desktop able to accept web CWTS data already? If so, great, if not, we'll have to disable CWTS for the firstrun flow, and when the Fx patch lands, enable CWTS for firstrun, maybe using a different context?

vladikoff commented on app/scripts/models/auth_brokers/first-run.js:
```diff
@@ -23,6 +24,11 @@ define([
+    defaultCapabilities: _.extend({}, proto.defaultCapabilities, {
+      chooseWhatToSyncCheckbox: true,
+      chooseWhatToSyncWebV1: false
+    }),
```
From @shane-tomlinson feedback, we disabled CWTS in first run. 

Also this branch has the old version once we are ready to enable CWTS in first run https://github.com/mozilla/fxa-content-server/tree/cwts-firstrun-future

vladikoff commented:
I addressed the feedback ( https://github.com/mozilla/fxa-content-server/pull/3214#discussion_r43286298 ) and merging this now to test sync using latest.dev.lcip.org

Shane feel free to recommend tweaks and I'll add them in future PRs
